app/src/main/res/layout/activity_main.xml

```diff
@@ -10,7 +10,6 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical"
-        android:paddingTop="@dimen/activity_vertical_margin"
```

Comment:
you should mention that in the decription whatevr changes you make, please follow best practices.